docs/pages/upgrading/agent-managed-updates.mdx

```diff
@@ -559,3 +559,37 @@ $ journalctl -u teleport-update
    $ sudo teleport-update update --now
    ```
 
+### Using a different CDN URL
+
+If your agents cannot reach the default Teleport CDN URL (`cdn.teleport.dev`), they will be unable to download updates.
+
+Here are a couple of potential solutions to this issue:
+
+#### Use an HTTP CONNECT proxy
```

Comment:
Might consider adding a section for terminating proxies (`HTTP_PROXY`). Those may be a bit more common?

Reply:
IIRC `HTTP_PROXY` is used for queries to URLs starting `http://` whereas `HTTPS_PROXY` is used for `https://` (as all standard `teleport-update` CDN/proxy-bound traffic would be) - it seemed a little redundant to set it, because we don't expect any HTTP traffic to originate from the binary.

One exception would be if someone overrides `base-url` to be `http://`-based and _also_ needs to use an HTTP CONNECT proxy to get to that base URL.